### PR TITLE
WIP: Doc: Add Windows worker installation documentation

### PR DESCRIPTION
**Pull request by pmisik**
Relates to #6048

## Remove this paragraph

If you don't remove this paragraph from the pull request description, this means you didn't read our contributor documentation, and your patch will need more back and forth before it can be accepted!

Please have a look at our developer documentation before submitting your Pull Request.

http://docs.buildbot.net/latest/developer/quickstart.html

And especially:
http://docs.buildbot.net/latest/developer/pull-request.html

## Contributor Checklist:

* [ ] I have updated the unit tests
* [ ] I have created a file in the `master/buildbot/newsfragments` directory (and read the `README.txt` in that directory)
* [ ] I have updated the appropriate documentation

### CODE REVIEW

File: master/docs/manual/installation/misc.rst [hunk 1]
```diff
@@ -79,9 +79,100 @@ With a little modification, these scripts can be used on both Debian and RHEL-ba
...
+```
+
+Troubleshooting
+===============
+If anything goes wrong check
+- twisted log on C:\bbw\worker\twistd.log
+- Windows system event log.
+
 You can find information about installation of Buildbot as Windows service in
 `RunningBuildbotOnWindows <http://trac.buildbot.net/wiki/RunningBuildbotOnWindows>`_.
```
Comment (levitte) [inline master/docs/manual/installation/misc.rst:175]: Perhaps remove these two lines, since they aren't relevant any more?

**Comment by levitte** [comment]
I plan on trying these instructions in a few days

**Review by tardyp — COMMENTED** [review]
looks like very useful! thanks!

**Comment by pmisik** [comment]
@tardyp 

Are you able recover original trac document on http://trac.buildbot.net/wiki/RunningBuildbotOnWindows?

**Comment by tardyp** [comment]
web archive to the rescue..
https://web.archive.org/web/20201113084744/http://trac.buildbot.net/wiki/RunningBuildbotOnWindows

**Review by p12tic — APPROVED** [review]

**Comment by p12tic** [comment]
Looks great, thank you. I've added an exception to the spelling word list, rebased and will merge soon.

**Pull request merged**